.circleci/config.yml

```diff
@@ -656,6 +656,23 @@ jobs:
           paths:
             - rediscluster.results
       - *save_cache_step
+      
+  vertica:
+    docker:
+      - *test_runner
+      - image: sumitchawla/vertica
+        env:
+          - VP_TEST_USER=dbadmin
+          - VP_TEST_PASSWORD=abc123
+          - VP_TEST_DATABASE=test
+    steps:
+      - checkout
+      - *restore_cache_step
+      - run: tox -e 'vertica_contrib-{py27,py34,py35,py36}-vertica{060,070}' --result-json /tmp/vertica.results
+      - persist_to_workspace:
+          root: /tmp
+          paths:
+            - vertica.results
```

Comment:
Here we should `- *save_cache_step` for the next runs.

Reply:
Ahhh! Lost in the merge conflict!